src/Game.cpp

```diff
@@ -67,88 +71,97 @@ Game::Game(Arguments&& args)
 	}
 	_renderer = std::make_unique<Renderer>(_window.get(), args.rendererType, args.vsync);
 	_fileSystem->SetGamePath(GetGamePath());
+	_handModel = std::make_unique<L3DMesh>();
+	_handModel->LoadFromFile("Data/CreatureMesh/Hand_Boned_Base2.l3d");
```

Comment:
There's a question we have to answer here.
Should the test model also be drawn through the entity registry?
The sky?
The Land?

Reply:
absolutely - all should probably have a MeshRenderer class that provides the various parameters required for bgfx submit / state